src/app/src/components/QuizQuestion.js

```diff
@@ -0,0 +1,89 @@
+import PropTypes, { func } from 'prop-types';
+import React from 'react';
+import styled from 'styled-components';
+import { Button, Flex } from 'rebass';
+import { themeGet } from 'styled-system';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+import QuizOption from './QuizOption';
+
+import { QuizFish } from '../util/QuizFish';
+
+const StyledQuizQuestion = styled(Flex)`
+    text-align: center;
+    background: ${themeGet('colors.teals.0')};
+    flex-direction: column;
+`;
+
+const Options = styled(Flex)`
+    flex-direction: row;
+    flex-wrap: wrap;
+    text-align: center;
+`;
+
+const StyledQuizOption = styled(QuizOption)`
+    width: 50%;
+    height: 50%;
+    margin: auto;
+`;
+
+class QuizQuestion extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+            guessed: [],
+            usedHint: false,
+        };
+    }
+
+    render() {
+        const { answer, choices, onDone } = this.props;
+        const { guessed, usedHint } = this.state;
+
+        const showHint = guessed.length > 0 || usedHint;
+        // TODO: Remove the default value here when we get hint text
```

Comment:
@fungjj92 it doesn't look like we have an issue already for the hint text, should I open one?

Reply:
Yeah! Put a blocked tag please.